src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -26,8 +26,10 @@ internal static class DependentProjectsFinder
         /// </summary>
         private struct DefinitionProject
         {
+#pragma warning disable IDE0052 // Remove unread private members
```

Comment:
❕ This one needs to be fixed instead of suppressed.

Reply:
I am hesitant to make any non-cleanup/non-style changes in this PR. I will file a tracking bug for a follow-up PR.

Reply:
https://github.com/dotnet/roslyn/issues/43406